packages/idos-sdk-js/src/lib/data.ts

```diff
@@ -1,13 +1,14 @@
 import * as Base64Codec from "@stablelib/base64";
-import { idOS } from "./idos";
+import type { Enclave } from "./enclave";
+import type { KwilWrapper } from "./kwil-wrapper";
 
 /* global crypto */
 
 export class Data {
-  idOS: idOS;
+  constructor(public readonly kwilWrapper: KwilWrapper, public readonly enclave: Enclave) {}
 
-  constructor(idOS: idOS) {
-    this.idOS = idOS;
+  static async init(kwilWrapper: KwilWrapper, enclave: Enclave) {
+    return new Data(kwilWrapper, enclave);
```

Comment:
Let's remove the init, since it doesn't do any additional work over the constructor.